.stylelintignore

```diff
@@ -0,0 +1,6 @@
+# Node modules
+node_modules/
```

Comment:
It should be in default ignore

Reply:
What do you mean by «default ignore»?

Reply:
> Stylelint ignores the node_modules directory by default. 

https://stylelint.io/user-guide/configure/#ignorefiles